project-communications/participation-communication-strategy.md

```diff
@@ -0,0 +1,131 @@
+# Participation and Communication Strategy
+
+This is where you will find all information and strategic direction of our communicaion and engagement to ensure participation on the AutSPACEs Project. 
+
+## Mountain of Engagement
+
+Our Mountain of Engagement (MoE) can be seen in the below image. 
+
+[insert image here]
+
+It shows the levels of engagement and participation that there is within the Project. 
+It is important to note that there is not a hierarchy in relation to how important you are to the project. 
+Each level is crucial and fundamental to the success of the project. 
+
+The MoE allows us to map out where contributors lie and where we can provide opportunities to allow individuals to increase their participation to the next level. 
+A key difference between each level is the responsibility the individual has to the project. 
+As the level increases so does that individuals responsibility accordingly. 
+
+The MoE seeks out opportunities for contrbutors to move up the scale if they are happy to do so and the community manager will ensure this happens and opportunities are available. 
+
+## Onboarding New Members
+
+When new members show interest in being a part of the project, they will be onboarded and briefed on the project's aims and objectives as well as progress update. 
+This will be in the form of an onboarding pack which includes all materials that is needed to succesfully understand the project, as well as a 1:1 chat with the Community Manager who will ensure all information is understood. 
```

Comment:
```suggestion
This will be in the form of an onboarding pack which includes all materials that are needed to succesfully understand the project, as well as a 1:1 chat with the Community Manager who will ensure all information is understood. 
```